src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -226,6 +226,59 @@ class CosmasGoogleCloudService(private val bucketName: String,
         responseObserver.onCompleted()
     }
 
+    override fun deleteFile(request: CosmasProto.DeleteFileRequest, responseObserver: StreamObserver<CosmasProto.DeleteFileResponse>) {
+        LOG.info("Get request for delete file \"${request.fileName}\" # ${request.fileId}")
+        val cemeteryName = request.projectId
```

Comment:
Let's add "-cemetery" suffix, just to be able to distinguish these blobs when scanning with eyes or Ctrl+F

Reply:
done